docs/images/metro/txt2image.md

```diff
@@ -1,21 +0,0 @@
-# Install desktop app
```

Comment:
Why this deletion ?
I personnaly found it useful when updating all the images.

Reply:
It was a comment by the reviewer.

Reply:
Okay I will keep it separately ^^